pkg/controller/codestatus/show.go

```diff
@@ -34,15 +34,24 @@ func (c *Controller) HandleShow() http.Handler {
 		}
 		flash := controller.Flash(session)
 
-		uuids := r.URL.Query()["uuid"]
-		var uuid string
-		if len(uuids) == 1 {
-			uuid = uuids[0]
+		realm := controller.RealmFromContext(ctx)
+		if realm == nil {
+			controller.MissingRealm(w, r, c.h)
+			return
 		}
+
 		m := controller.TemplateMapFromContext(ctx)
-		m["UUID"] = uuid
+		type FormData struct {
```

Comment:
Move this to before the HandlerFunc (but still in HandleShow) so it's only created once.